tests/AntiMattr/Tests/MongoDB/Migrations/Configuration/ConfigurationTest.php

```diff
@@ -160,14 +151,8 @@ public function testGetNumberOfExecutedMigrations()
             ->with('antimattr_migration_versions_test')
             ->willReturn($collection);
 
-        $cursor = $this->createMock('Doctrine\MongoDB\Cursor');
-
         $collection->expects($this->once())
-            ->method('find')
-            ->willReturn($cursor);
-
-        $cursor->expects($this->once())
-            ->method('count')
+            ->method('countDocuments')
```

Comment:
This methods exists in mongodb/mongodb only from version >= 1.4.0. I think that we should require in composer.json "mongodb/mongodb": "^1.4".